src/core/project/project.ts

```diff
@@ -433,23 +433,26 @@ export class Project {
     this.resetActivities();
 
     this._simulation.start().then((response: any) => {
-      if (response == null || response.status != 200 || response.data == null) {
+      if (
+        response == null ||
+        response.status != 200 ||
+        response.data == null ||
+        response.data.hasOwnProperty('data')
```

Comment:
You probably want this instead: :wink: 
```suggestion
        !response.data.hasOwnProperty('data')
```